scripts/dataset/index.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import generateRelease from './export/index.js';
+import logger from './logger/index.js';
+import publishRelease from './publish/index.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+export async function release({ publicationEnabled, removeLocalCopyEnabled, fileName }) {
```

Comment:
Use consistent flag names: either `publicationEnabled` and `localCopyRemovalEnabled`, or `publishEnabled` and `removeLocalCopyEnabled`. Or maybe `shouldPublish` and `shouldRemoveLocalCopy`? 🙂

Reply:
Go for `should…`.